frontend/src/metabase/core/components/Button/Button.styled.tsx

```diff
@@ -11,6 +11,12 @@ export const ButtonRoot = styled.button<ButtonRootProps>`
   transition: all 200ms linear;
   flex-shrink: 0;
 
+  &:hover {
+    /* XXX: standardize button transition, it wouldn't make much sense to only extend this in a single place, since transition should be a part of the design system. */
+    transition-property: color, border-color, background-color, opacity,
+      visibility;
```

Comment:
I want to hear @mazameli 's opinion on this. I think we've had a couple of similar discussions about whether a certain style should be scoped to a specific UI components, or should that be baked into the code components.

From a design's point of view, I couldn't think of a way to want to only have a few buttons that would fade in and out while most of the other buttons instantly appear and disappear since that creates inconsistency in the design.

Let me know if fading in and out buttons should only be scoped to the **Apply** button specifically for this project.